dev-helpers/template.ejs

```diff
@@ -67,37 +66,37 @@
 
 <div id="swagger-ui"></div>
 
-<script src="./swagger-ui-bundle.js"> </script>
```

Comment:
I see this was removed; where is the bundle and standalone preset coming from now that this is gone?